lxd/devlxd.go

```diff
@@ -175,7 +175,11 @@ var devlxdEventsGet = devLxdHandler{"/1.0/events", func(d *Daemon, c instance.In
 
 var devlxdAPIHandler = devLxdHandler{"/1.0", func(d *Daemon, c instance.Instance, w http.ResponseWriter, r *http.Request) response.Response {
 	if r.Method == "GET" {
-		location := "none"
+		location, err := os.Hostname()
```

Comment:
Can we do this in a way so that if we are clustered we dont make an unnecessary syscall to os.Hostname()?

Reply:
Absolutely - fixed now :)